libs/crypto/integer.hpp

```diff
@@ -14,10 +14,12 @@
 
 #pragma once
 
+#ifdef USE_OPENSSL
```

Comment:
this should be under openssl in the directory layout as there is no other implementation